src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Providers/AzureWorkloadProviderHelper.cs

```diff
@@ -194,23 +194,12 @@ public AzureWorkloadProviderHelper(ServiceClientAdapter serviceClientAdapter)
         {
             string vaultName = (string)providerData[CmdletModel.VaultParams.VaultName];
             string vaultResourceGroupName = (string)providerData[CmdletModel.VaultParams.ResourceGroupName];
-            string name = (string)providerData[CmdletModel.ContainerParams.Name];
```

Comment:
See the file change above - remove this file 'set'

Reply:
Done